src/libraries/System.Text.Json/src/System/Text/Json/Node/JsonNode.To.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers;
+
+namespace System.Text.Json.Node
+{
+    public abstract partial class JsonNode
+    {
+        /// <summary>
+        ///   Converts the current instance to string in JSON format.
+        /// </summary>
+        /// <param name="options">Options to control the serialization behavior.</param>
+        /// <returns>JSON representation of current instance.</returns>
+        public string ToJsonString(JsonSerializerOptions? options = null)
+        {
+            var output = new ArrayBufferWriter<byte>();
+            using (var writer = new Utf8JsonWriter(output, options == null ? default(JsonWriterOptions) : options.GetWriterOptions()))
+            {
+                WriteTo(writer, options);
+            }
+            return JsonHelpers.Utf8GetString(output.WrittenSpan);
+        }
+
+        /// <summary>
+        ///   Gets a string representation for the current value appropriate to the node type.
+        /// </summary>
+        /// <returns>A string representation for the current value appropriate to the node type.</returns>
+        public override string ToString()
+        {
+            // Special case for string; don't quote it.
```

Comment:
Curious why this is? Based on the rest of the method body I would have expected the output to be valid JSON.

Reply:
Per review, this was requested by the Azure SDK team. These values are supposed to be human-readable and do not need to be round-trippable JSON. Plus it includes whitespace that normally users don't want for performance.

The `ToJsonString()` should be used to obtain round-trippable JSON.

Reply:
Got it, thanks.